packages/notification-center/src/utils/react-to-webcomponent.ts

```diff
@@ -99,6 +100,10 @@ export default function (
   ReactComponent: React.FunctionComponent<any> | React.ComponentClass<any>,
   options: R2WCOptions = {}
 ): CustomElementConstructor {
+  if (isSSR) {
+    return;
+  }
```

Comment:
we are in the react to web component proxy file
when it's used during SSR just do nothing, because the Web APIs do not exist on server side